ts-lib/tests/local.test.ts

```diff
@@ -0,0 +1,62 @@
+import { Worker, NearAccount } from "near-workspaces";
+
+describe("EthKeys contract tests", () => {
+  let worker: Worker;
+  let user: NearAccount;
+  let contract: NearAccount;
+
+  beforeEach(async () => {
+    // Init the worker and start a Sandbox server
+    worker = await Worker.init();
+
+    // Deploy contract
+    user = worker.rootAccount;
+    contract = await user.createSubAccount("test-account");
+
+    // Get wasm file path from package.json test script in folder above
+    const wasmFilePath = "../target/wasm32-unknown-unknown/debug/neareth.wasm";
+    await contract.deploy(wasmFilePath);
+  });
+  afterEach(async () => {
+    // Stop Sandbox server
+    await worker.tearDown();
+  });
+
+  it("get_key", async () => {
+    const key = await contract.view("get_key", { account_id: user.accountId });
+    expect(key).toBe(null);
+  });
+  it("set_key", async () => {
+    const value = "Howdy";
+    await user.call(contract, "set_key", {
+      encrypted_key: value,
+      overwrite: false,
```

Comment:
I would consider making this optional as it's gonna be false in most cases.

Reply:
Do you mean to make the Contract's input parameter optional or the ts-library functions?

Reply:
Fixed in 726d486